src/main/java/sk/uniba/fmph/dcs/player_board/PlayerBoardFacade.java

```diff
@@ -0,0 +1,93 @@
+package sk.uniba.fmph.dcs.player_board;
+
+import sk.uniba.fmph.dcs.stone_age.Effect;
+import sk.uniba.fmph.dcs.stone_age.EndOfGameEffect;
+import sk.uniba.fmph.dcs.stone_age.InterfaceFeedTribe;
+import sk.uniba.fmph.dcs.stone_age.InterfaceNewTurn;
+import sk.uniba.fmph.dcs.stone_age.InterfacePlayerBoardGameBoard;
+
+import java.util.Collection;
+import java.util.OptionalInt;
+
+public final class PlayerBoardFacade implements InterfaceFeedTribe, InterfaceNewTurn, InterfacePlayerBoardGameBoard {
+    private final PlayerBoard playerBoard;
+
+    public PlayerBoardFacade() {
+        this.playerBoard = new PlayerBoard();
+    }
+
+    @Override
+    public boolean feedTribeIfEnoughFood() {
+        return playerBoard.feedTribeIfEnoughFood();
+    }
+
+    @Override
+    public boolean feedTribe(final Collection<Effect> resources) {
+        return playerBoard.feedTribe(resources);
+    }
+
+    @Override
+    public boolean doNotFeedThisTurn() {
+        return playerBoard.doNotFeedThisTurn();
```

Comment:
```suggestion
        boolean ret = playerBoard.tribeFedStatus().setTribeFed();
        if (!ret) {
            playerBoard.takePoints(PlayerBoard.POINTS_TO_TAKE_IF_TRIBE_IS_NOT_FED);
        }
        return ret;
```
Funkcionalita presunutá z `PlayerBoard`.